apps/editors/helpers.py

```diff
@@ -476,8 +478,13 @@ def get_actions(self, request, addon):
         labels, details = self._review_actions()
 
         actions = SortedDict()
-        if not addon.admin_review or acl.action_allowed(
-                request, 'ReviewerAdminTools', 'View'):
+        if ((not addon.admin_review or acl.action_allowed(request,
+             'ReviewerAdminTools', 'View'))
+            and (not GroupUser.objects.filter(group=Group.objects.get(
+                 name='Limited Reviewers'), user=request.user)
+                 or (datetime.datetime.now() -
+                     addon.latest_version.nomination > datetime.timedelta(
+                     hours=REVIEW_LIMITED_DELAY_HOURS)))):
```

Comment:
If I do that, flake8 complains:
`continuation line over-indented for hanging indent`